### PR TITLE
Updated env variables to the latest versions

### PR DESCRIPTION
**Pull request by idvoretskyi**
For simplifying the deployment process, the sample components versions were updated to the latest actual.

---

This change is [Reviewable](http://reviewable.k8s.io/reviews/kubernetes/kubernetes/24386)

### CODE REVIEW

**Comment by brendandburns** [comment]
Does this etcd version actually work?  I'm not sure we've pushed the new one to gcr.io.

Thanks
--brendan

**Comment by brendandburns** [comment]
Also, please see the changes in how docs are updated:

See http://kubernetes.io/editdocs/ for information about editing docs

**Comment by idvoretskyi** [comment]
@brendandburns let's put the PR on hold - it has to be tested deeper and the results will be provided.

**Comment by idvoretskyi** [comment]
Referring to https://github.com/kubernetes/kubernetes/pull/24387 this PR has to be closed.

@brendandburns is there any possibility the find list of etcd versions, placed on `gcr.io`?

File: docs/getting-started-guides/docker-multinode/master.sh [hunk 1]
```diff
@@ -26,8 +26,8 @@ if ( ! ps -ef | grep "/usr/bin/docker" | grep -v 'grep' &> /dev/null ); then
 fi
 
 # Make sure k8s version env is properly set
-K8S_VERSION=${K8S_VERSION:-"1.2.0-alpha.7"}
-ETCD_VERSION=${ETCD_VERSION:-"2.2.1"}
+K8S_VERSION=${K8S_VERSION:-"1.2.2"}
+ETCD_VERSION=${ETCD_VERSION:-"2.3.1"}
```
Comment (thockin) [inline docs/getting-started-guides/docker-multinode/master.sh]: I don't think we should bump etcd up right now.  We keep our own build on purpose, so if we feel we need 2.3, we should rebuild it for the architectures we support and use it everywhere.

File: docs/getting-started-guides/docker-multinode/master.sh [hunk 2]
```diff
@@ -26,8 +26,8 @@ if ( ! ps -ef | grep "/usr/bin/docker" | grep -v 'grep' &> /dev/null ); then
 fi
 
 # Make sure k8s version env is properly set
-K8S_VERSION=${K8S_VERSION:-"1.2.0-alpha.7"}
-ETCD_VERSION=${ETCD_VERSION:-"2.2.1"}
+K8S_VERSION=${K8S_VERSION:-"1.2.2"}
```
Comment (thockin) [inline docs/getting-started-guides/docker-multinode/master.sh]: this is stale now :)

File: docs/getting-started-guides/docker-multinode/master.sh (hunk 1 above)
Comment (idvoretskyi) [inline docs/getting-started-guides/docker-multinode/master.sh]: @thockin how the customers may clarify what the etcd versions are available to use?

**Comment by idvoretskyi** [comment]
Closed an non-actual.

**Pull request closed without merging**